src/adapters/handlers/catalog.ts

```diff
@@ -16,9 +16,7 @@ export function createCatalogHandler(
     const params = new Params(context.url.searchParams)
     const onlyListing = params.getBoolean('onlyListing')
     const onlyMinting = params.getBoolean('onlyMinting')
-    const sortBy =
-      params.getValue<CatalogSortBy>('sortBy', CatalogSortBy) ||
-      CatalogSortBy.CHEAPEST
+    const sortBy = params.getValue<CatalogSortBy>('sortBy', CatalogSortBy)
```

Comment:
This should not set a default filter, it's already handled at the query level when it doesn't get any `sortBy` value.